src/index.html

```diff
@@ -13,6 +13,40 @@
     />
```

Comment:
The href attribute in your link tag should point to a CSS file, not a SCSS file. SCSS files need to be compiled into regular CSS. The correct link should be `href="styles/index.css"`.